src/Sylius/Bundle/ReviewBundle/Resources/translations/messages.en.yml

```diff
@@ -0,0 +1,21 @@
+# This file is part of the Sylius package.
+# (c) Paweł Jędrzejewski
+
+sylius:
+    review:
+        header: Product reviews
+        no_results: There are no reviews for this product.
```

Comment:
Why ReviewBundle would know about ``product`` translations? This should be removed from here. Only ``form`` part should stay.